src/google.py

```diff
@@ -45,10 +46,10 @@ def login():
     Returns:
         json: user_info.
     """
-       
+     
     if 'id_token' not in request.json:
         return jsonify({
-            'error': "'id_token' is missing from request"
+            'error': "id_token is missing from request"
```

Comment:
Please be consistent with quotes. Are you using single quotes or double quotes?

Same comment for the rest of the changes in this PR.

Reply:
??

Reply:
The key have single quotes, while the values are double quotes, incase the values may have `'`